openedx/features/ucsd_features/tests/test_utils.py

```diff
@@ -21,7 +21,7 @@
 
 @ddt.ddt
 class UCSDFeaturesUtilsTests(ModuleStoreTestCase):
-    """ Tests for the utils used by ucsd-specific customizations """
+    """ Tests for the utils used by ucsd-specific specific customizations """
```

Comment:
kindly follow one convention to write doc strings

like

""" <br>
doc string here <br>
"""

Reply:
This file wasn't supposed to be changed in this PR. I did it by mistake.
But have changed the docstring as you suggested.